pop quizzes added to mux-functions.md (#510)
CircuitVerse/Interactive-Book: docs/comb-msi/mux-functions.md

## References

{% bibliography --cited --file books --file mux %}

{:.quiz}

  1. Can one MUX block be said an universal logic block ?
     1. Yes
     * No
  2. Can one DEMUX block be said an universal logic block ?
     1. Yes
     * No

>   3. Which of the following option is correct regarding MUX-based gates ?

Reviewer: Question can be rephrased as something like "Select the MUX-based gates which is/are possible"

>   3. Which of the following option is correct regarding MUX-based gates ?

Author: ```suggestion
  3. Select the MUX-based gates which is/are possible
```

>      * MUX-based XOR gate is impossible

Reviewer: is impossible in answers is not required
```suggestion
     * MUX-based XOR gate is impossible
```
